### PR TITLE
Update node to 16.14 from 16.10

### PR DESCRIPTION
**Pull request by visargsoneji**

### CODE REVIEW

File: broker/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # FROM node:14.16-alpine3.13 also works here for a smaller image
-FROM node:16.10-alpine3.14
+FROM node:16.14-alpine3.14
```
Comment (anoopjb) [inline broker/Dockerfile:2]:
```suggestion
FROM node:16.14-alpine3.15
```
I think alpine 3:15 is available.

Comment (visargsoneji) [reply broker/Dockerfile]: Done

**Review by anoopjb — APPROVED** [review]

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n+n+n failed.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-2 is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-1 is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n+n+n is succeeded.

**Pull request merged**